src/ripple/app/ledger/impl/LedgerMaster.cpp

```diff
@@ -292,8 +295,9 @@ class LedgerMasterImp
         {
             // Calculate the sample median
             std::sort (times.begin (), times.end ());
-            signTime = (times[times.size() / 2] +
-                times[(times.size() - 1) / 2]) / 2;
+            auto const t0 = times[(times.size() - 1) / 2];
+            auto const t1 = times[times.size() / 2];
+            signTime = t0 + (t1 - t0)/2;
```

Comment:
The rationale for this change is to keep the units consistent.  The type of `times[times.size() / 2]` and `times[(times.size() - 1) / 2]` is `NetClock::time_point` and it is a compile-time error to add two `time_points` together.  You can only subtract them which results in a `NetClock::duration`.  This is much like pointer arithmetic.  You can't add two pointers, only subtract them to get a `ptrdiff_t`.  If motivated, we could escape the type system here with `.time_since_epoch().count()`, but I didn't see the motivation to do that.